cf-nodejs-logging-support-core/log-core.js

```diff
@@ -716,6 +736,37 @@ var overrideField = function (field, value) {
     return false;
 };
 
+//sets
+var overrideCustomFieldFormat = function(value) {
```

Comment:
Is this function used only for testing?

Reply:
The idea was to maybe expose this to the customers to enforce a specific logging format, however I am not sure if this is needed.